gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/06_add_scoring_table.yml

```diff
@@ -20,6 +20,15 @@ databaseChangeLog:
               - column: { name: rule, type: nvarchar(64), constraints: { nullable: true } }
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.